met-web/src/components/common/index.tsx

```diff
@@ -47,6 +47,25 @@ export const StyledSecondaryButton = styled(MuiButton)(() => ({
     },
 }));
 
+export const StyledNeutralButton = styled(MuiButton)(() => ({
```

Comment:
I read in #1027 that MA called them WidgetButtons. Our naming basically defines how we are going to use them. So maybe we can call them WidgetButtons, a quick check with MA would be great about where this style is used

Reply:
Sure thing ill see what she says